portfolio/src/main/webapp/2020.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>sehhaj portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+        <h1>sehhaj's portfolio - still to be implementated</h1>
+    </div>
+  </body>
+
+  <div id="nav-bar">
+    <a href="index.html">WELCOME HOME</a>
+    <a href="about.html">ABOUT ME</a>
+    <a href="gallery.html">ARCHIVE GALLERY</a>
+    <a href="2020.html">BEST OF 2020</a>
+    <a href="procrastination.html">THE PROCRASTINATION CORNER</a>
+    <a href="blog.html">BLOG POSTS</a>
+  </div>
+</html>
```

Comment:
include a blank new line at the end of your files, this is a little bit of a historic thing because bash commands like cat and grep always expected there to be a new line at the end of the file, but you still find odd tools that expect it so it's become command practice. (please make sure this is true for other files too).

Reply:
I did update the code to reflect this. On my local branch I have all the blank new lines at the end of my files. However  I just noticed that the files I committed on github don't have the new line I pushed?